### PR TITLE
Add SQL Warehouse to Spark integrations

### LINKED ISSUE #6938: Document how to use Databricks SQL Warehouse with lakeFS

### PR DESCRIPTION
**Pull request by itaiad200**
closes #6938

### CODE REVIEW

File: docs/integrations/spark.md [hunk 1]
```diff
@@ -390,6 +390,28 @@ spark.hadoop.fs.lakefs.endpoint=‘https://example-org.us-east-1.lakefscloud.io
...
+A SQL warehouse is a compute resource that lets you run SQL commands on data objects within Databricks SQL
+
+If you use Databricks SQL warehouse, you can take advantage of the lakeFS 
+S3-compatible API with the S3A FileSystem. 
+
+Define your SQL Warehouse configurations in the following way:
+
+1. In the top right, select `Admin Settings` and then `SQL warehouse settings`.
+
+2. Under `Data Access Configuration` add the following key-value pairs:
```
Comment (talSofer) [inline docs/integrations/spark.md:404]: What would be the style of URIs the user should use in this case? based on the configurations I guess that the scheme is lakefs//:. If this is the case let's add an example usage

**Review by talSofer — CHANGES_REQUESTED** [review]
Thanks! please see my question, if I got it incorrectly let me know and I'll drop it :)

File: docs/integrations/spark.md [hunk 2]
```diff
@@ -390,6 +390,28 @@ spark.hadoop.fs.lakefs.endpoint=‘https://example-org.us-east-1.lakefscloud.io
 
 For more details about [Mounting cloud object storage on Databricks](https://docs.databricks.com/dbfs/mounts.html).
 
+### Configuring Databricks SQL Warehouse with the S3-compatible API
+
+A SQL warehouse is a compute resource that lets you run SQL commands on data objects within Databricks SQL
```
Comment (arielshaqed) [inline docs/integrations/spark.md:395]: Nit:
```suggestion
A SQL warehouse is a compute resource that lets you run SQL commands on data objects within Databricks SQL.
```

File: docs/integrations/spark.md (hunk 1 above)
Comment (itaiad200) [reply docs/integrations/spark.md]: Good thing someone reads what I wrote here 😄

**Review by talSofer — APPROVED** [review]

**Pull request merged**